package.json

```diff
@@ -102,6 +102,7 @@
     "audit": "yarn npm audit --recursive --environment production --severity moderate"
   },
   "resolutions": {
+    "@metamask/network-controller@^12.0.0": "npm:@metamask-previews/network-controller@12.1.2-preview.0f6a2fb",
```

Comment:
Which PR/branch is this preview build coming from?

Reply:
https://github.com/MetaMask/core/pull/1557

Reply:
above PR merged, waiting on release in core now